src/Microsoft.Identity.Client/AppConfig/PublicClientApplicationBuilder.cs

```diff
@@ -132,6 +132,21 @@ public PublicClientApplicationBuilder WithUseCorporateNetwork(bool useCorporateN
         }
 #endif
 
+#if RUNTIME || NETSTANDARD_BUILDTIME
+        /// <summary>
+        ///  Sets a reference to the ViewController (if using Xamarin.iOS), Activity (if using Xamarin.Android)
+        ///  IWin32Window or IntPtr (if using .Net Framework). Used for invoking the browser.
+        /// </summary>
+        /// <remarks>Mandatory only on Android. Can also be set via the AcquireTokenInteractive builder.</remarks>
+        /// <param name="parentActivityOrWindowFunc">The parent as an object, so that it can be used from shared NetStandard assemblies</param>
+        /// <returns>The builder to chain the .With methods</returns>
+        public PublicClientApplicationBuilder WithParentActivityOrWindow(Func<object> parentActivityOrWindowFunc)
```

Comment:
maybe it's my lack of mastering the English language, but "Func" looks weird to me. Wouldn't want to use Callback in the public API ?